etc/plf/setenv-customize.sh

```diff
@@ -61,6 +61,15 @@ if ${DEPLOYMENT_CHAT_ENABLED}; then
     CATALINA_OPTS="${CATALINA_OPTS} -Dchat.dbName=${EXO_CHAT_MONGODB_NAME}"
     CATALINA_OPTS="${CATALINA_OPTS} -Dchat.chatPassPhrase=${EXO_CHAT_MONGODB_NAME}"
 fi
+
+# eXo Onlyoffice addon
+if ${DEPLOYMENT_ONLYOFFICE_DOCUMENTSERVER_ENABLED}; then
+    CATALINA_OPTS="${CATALINA_OPTS} -Donlyoffice.documentserver.host=localhost:${DEPLOYMENT_ONLYOFFICE_HTTP_PORT}"
+    CATALINA_OPTS="${CATALINA_OPTS} -Donlyoffice.documentserver.schema=http"
+    CATALINA_OPTS="${CATALINA_OPTS} -Donlyoffice.documentserver.allowedhosts=localhost,${INSTANCE_KEY}.${ACCEPTANCE_HOST}"
```

Comment:
is it working if we specify a virtual host with `DEPLOYMENT_APACHE_VHOST_ALIAS ` ?

Reply:
DEPLOYMENT_EXT_HOST or DEPLOYMENT_APACHE_VHOST_ALIAS work fine, but in dev mode(adt-dev.sh) they get localhost or empty.
I used ${INSTANCE_KEY}.${ACCEPTANCE_HOST} to circumvent this part (L1136-1142 of functions.sh) : 
  if ${ADT_DEV_MODE}; then
    env_var "DEPLOYMENT_EXT_HOST" "localhost"
    env_var "DEPLOYMENT_EXT_PORT" "${DEPLOYMENT_HTTP_PORT}"
  else
    env_var "DEPLOYMENT_EXT_HOST" "${INSTANCE_KEY}.${ACCEPTANCE_HOST}"
    env_var "DEPLOYMENT_EXT_PORT" "80"
  fi

example : 
    CATALINA_OPTS="${CATALINA_OPTS} -Donlyoffice.documentserver.allowedhosts=localhost,${INSTANCE_KEY}.${ACCEPTANCE_HOST},${DEPLOYMENT_APACHE_VHOST_ALIAS}"


will give -Donlyoffice.documentserver.allowedhosts=localhost,plfent-5.1.0.acceptance.exoplatform.org,

I opted for the solution that works in dev or prod mode.

Reply:
I don't understand your response!

If we configure the acceptance instance with a custom virtual-host we use the `DEPLOYMENT_APACHE_VHOST_ALIAS=my-custom-url.exoplatform.org` variable then all requests in the browser will be `http://my-custom-url.exoplatform.org/...`and not from `http://${INSTANCE_KEY}.${ACCEPTANCE_HOST}/..` usual url.

Are you sure we don't have to declare `DEPLOYMENT_APACHE_VHOST_ALIAS` url in the `onlyoffice.documentserver.allowedhosts` parameter ?

Reply:
I get your point now : 
In case we define DEPLOYMENT_APACHE_VHOST_ALIAS, onlyoffice.documentserver.allowedhosts will have a wrong value

I suggest the following :
As onlyoffice.documentserver.allowedhosts allows multiple values, we can use both (${INSTANCE_KEY}.${ACCEPTANCE_HOST} and ${DEPLOYMENT_APACHE_VHOST_ALIAS})
this will return a working and secure configuration.

WDYT ?

Reply:
I think you can propose something in that direction ;-)